recipe/meta.yaml

```diff
@@ -1,34 +1,76 @@
 package:
-  name: arm-variant
-  version: 1.1.0
+  name: arm-variant-metapackages
+  version: 1.2.0
 
 build:
   number: 0
   noarch: generic
-  string: "{{ arm_variant_type }}"
-  track_features:
-    - arm_{{ arm_variant_type }}  # [arm_variant_type != arm_variant_type_default]
 
-test:
-  commands:
-    - exit 0
+outputs:
 
-about:
-  home: https://github.com/conda-forge/arm-variant-feedstock
-  license: BSD-3-Clause
-  license_family: BSD
-  license_file: LICENSE.txt
-  summary: 'A meta-package to select the ARM variant.'
-  description: |
-    There are a couple of ARM variants that differ based on architecture.
+  - name: arm-variant
+    build:
+      noarch: generic
+      string: "{{ arm_variant_type }}"
+      track_features:
+        - arm_{{ arm_variant_type }}  # [arm_variant_type != arm_variant_type_default]
+    test:
+      commands:
+        - exit 0
+    about:
+      home: https://github.com/conda-forge/arm-variant-feedstock
+      license: BSD-3-Clause
+      license_family: BSD
+      license_file: LICENSE.txt
+      summary: 'A meta-package to select the runtime ARM variant.'
+      description: |
+        Use this package to select which ARM variant to use at runtime. Use the 'cuda-nvcc-arm-variant-target' package in build requirements to select which ARM variant to compile for.
+
+        There are a couple of ARM variants that differ based on architecture.
+
+        Currently the main ones are:
+          * SBSA ( Server Base System Architecture ), which is used in clusters, cloud environments, etc.
+          * Tegra ( System on a Chip ), which is used in edge applications like mobile internet devices, streaming systems, etc.
+
+        Mainly these differ as they have 2 different `cudatoolkit`s. This package is used to select between them.
+
+        The default is SBSA. So Tegra users should make sure to change this if needed.
 
-    Currently the main ones are:
-      * SBSA ( Server Base System Architecture ), which is used in clusters, cloud environments, etc.
-      * Tegra ( System on a Chip ), which is used in edge applications like mobile internet devices, streaming systems, etc.
+  - name: cuda-nvcc-arm-variant-target
+    build:
+      noarch: generic
+      string: "{{ arm_variant_type }}"
+      track_features:
+        - arm_{{ arm_variant_type }}  # [arm_variant_type != arm_variant_type_default]
+      run_exports:
+        strong_constrains:
+          arm-variant * {{ arm_variant_type }}
```

Comment:
Should this run export be in the compiler package only/instead? Or is it best used here?